content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,205 @@
+   helps make for an easier and more accurate selection, via a drop-down menu in the third field,
+   instead of risking a typo or incorrect value in the text box field that accompanies the
+   :guilabel:`contains` operator.
+
+In this third field, enter the name of the desired sales team(s) that are to be included in the
+report. It is important for all :guilabel:`contains` argument values to be specific enough and
+spelled correctly as they exist in Odoo, otherwise this risks returning multiple (or zero) values.
+
+.. image:: lead_distribution_report/sales-team-location.png
+  :align: center
+  :alt: Use Sales Team to filter the location the lead is associated with.
+
+.. important::
+   By adding more than one rule to the form, a new option emerges at the top of the pop-up window
+   above all the filters, to specify whether :guilabel:`any` :icon:`fa-caret-down` :guilabel:`(down
+   caret)` or :guilabel:`all` :icon:`fa-caret-down` :guilabel:`(down caret)` of the conditions
+   should match. This distinction is important to set correctly, as it impacts the driving logic of
+   how the filters return data.
+
+Click the default :guilabel:`any` :icon:`fa-caret-down` :guilabel:`(down caret)` menu item and be
+sure the :guilabel:`all` :icon:`fa-caret-down` :guilabel:`(down caret)` option is chosen instead.
+This setting will **only** show records that match *all* the rules contained inside the form.
+
+.. _crm/track_leads/phone-number:
+
+Contact method
+~~~~~~~~~~~~~~
+
+.. note::
+   The instruction below is not necessary, however, it's highly recommended to add a set contact
+   value to the report's search criteria. A lot of spam, duplicate, or low quality leads can easily
+   be screened out of the report simply by adding either a set :guilabel:`Phone` or
+   :guilabel:`Email` rule.
+
+Add another :guilabel:`New rule` to the form and set the first field to the first field to
+:guilabel:`Phone`. Then, select :guilabel:`is set` from the drop-down menu in the second field.
+Selecting this operator **only** filters for records that have a phone number associated with the
+lead.
+
+Alternatively (or in addition to the above rule), click :guilabel:`New rule` and set the first field
+to :guilabel:`Email`. Then, select :guilabel:`is set` from the drop-down menu in the second field.
+
+These rules add only leads with an associated contact method to the report.
+
+.. _crm/track_leads/active-status:
+
+Active status
+~~~~~~~~~~~~~
+
+Click the :icon:`fa-sitemap` :guilabel:`(Branch)` icon to the right of the `Phone is set` line, to
+add a new rule that branches from the rules above.
+
+Two horizontal sets of fields appear below a line showing :guilabel:`any of:` with a
+:icon:`fa-caret-down` :guilabel:`(down caret)` icon. This setting filters for
+records that match **any** of the rules contained inside. This uses the same logic as an OR (|)
+logical operator.
```

Comment:
Remove icon descriptors on the icon elements that are accompanied by text.

Suggesting to add `|` literal formatting to the OR operator, just because it looks better IMO 😄
```suggestion
Two horizontal sets of fields appear below a line showing :guilabel:`any` :icon:`fa-caret-down`
:guilabel:`of:` option. This setting filters for records that match **any** of the rules contained
inside. This uses the same logic as an OR (`|`) logical operator.
```